tests/providers/snowflake/hooks/test_snowflake.py

```diff
@@ -388,3 +388,38 @@ def test_connection_failure(self, mock_run):
             assert status is False
             assert msg == 'Connection Errors'
             mock_run.assert_called_once_with(sql='select 1')
+
+    @mock.patch('airflow.providers.snowflake.hooks.snowflake.SnowflakeHook.run')
+    def test_connection_without_region(self, mock_run):
```

Comment:
Can you add this case to the existing parameterized method?
https://github.com/apache/airflow/blob/e0552c7b4cf53757242c913fdb419ee55c70c459/tests/providers/snowflake/hooks/test_snowflake.py#L75-L203